sdk/storage/Azure.Storage.Blobs/tests/Azure.Storage.Blobs.Tests.csproj

```diff
@@ -10,7 +10,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="$(MSBuildThisFileDirectory)..\..\Azure.Storage.Common\tests\Shared\**\*.cs" />
-    <None Include="$(MSBuildThisFileDirectory)..\..\Azure.Storage.Common\tests\Shared\TestConfigurations.xml">
+    <None Include="$(MSBuildThisFileDirectory)..\..\Azure.Storage.Common\tests\Shared\*.xml">
```

Comment:
Motivation?

Reply:
There's no explicit ordering so the file may not have been created by the time this project builds.  (And we don't care at that point - if you don't have a config file, it doesn't matter if we copy it.)